installer/x86_64/install.sh

```diff
@@ -67,7 +67,12 @@ echo "onie_platform: $onie_platform"
 # default console settings
 CONSOLE_PORT=0x3f8
 CONSOLE_DEV=0
-CONSOLE_SPEED=9600
+
+# Pick up console speed from install enviroment, if failed, set it to 9600
+CONSOLE_SPEED=$(stty -F /dev/ttyS0 | grep speed | cut -d " " -f2)
```

Comment:
@lguohan thanks for the comments, will do further change.

Reply:
@lguohan I decide to get the console speed setting from '/proc/cmdline' instead of from ttyS*,  to avoid to check which ttyS is using.  please see the modification in the latest commit.